### ISSUE TITLE
steamvr-osvr install error

### ISSUE THREAD

**Issue by jdack41** [OSVR/Homebrew-OSVR#10]
When I was installing steam-osvr with homebrew, this error occurred.

==> Installing osvr/osvr/steamvr-osvr --HEAD
==> Cloning https://github.com/OSVR/SteamVR-OSVR.git
Updating ~/Library/Caches/Homebrew/steamvr-osvr--git
==> Checking out branch master
Synchronizing submodule url for 'vendor/docopt.cpp'
==> cmake .. -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG
Last 15 lines from ~/Library/Logs/Homebrew/steamvr-osvr/01.cmake:

  Could not find a package configuration file provided by "osvrRenderManager"
  with any of the following names:

    osvrRenderManagerConfig.cmake
    osvrrendermanager-config.cmake

  Add the installation prefix of "osvrRenderManager" to CMAKE_PREFIX_PATH or
  set "osvrRenderManager_DIR" to a directory containing one of the above
  files.  If "osvrRenderManager" provides a separate development package or
  SDK, be sure it has been installed.

-- Configuring incomplete, errors occurred!

**Comment by demonixis** [comment]
Hi,

You have to clone, compile and install RenderManager before trying to install steamVR from homebrew.

**Comment by jdack41** [comment]
Hi, demonixis.
I could install rendermanager, but I got same error with you.
https://github.com/OSVR/Homebrew-OSVR/issues/11

**Comment by demonixis** [comment]
Yes, I don't know why it's not working, it seems that @godbyk was able to compile it on its mac.

**Comment by jdack41** [comment]
Hmm... OK, thank you for your advice.
I tried to install from source, and it succeeded.
